Packs/EWS/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "EWS",
     "description": "Exchange Web Services and Office 365 (mail)",
     "support": "xsoar",
-    "currentVersion": "1.8.33",
+    "currentVersion": "1.8.34",
```

Comment:
maybe 1.9.0? as it is an announcement.

Reply:
yeah. there was an issue with the update-rn. fixing.